docs/manual/manual_buzzer_control.rst

```diff
@@ -0,0 +1,183 @@
+
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _manual_buzzer_control:
+
+********************
+Buzzer Controller
```

Comment:
I may accentuate the fact that this includes a bunch of song examples to help people find it. Maybe the title could be "Buzzer Controller and Status Songs" or something